proposals/lip-0061.md

```diff
@@ -534,15 +561,9 @@ verifyAggregateCommit(aggregateCommit):
     c = computeCertificateFromBlockHeader(blockHeader1)
     c.aggregationBits = aggregateCommit.aggregationBits
     c.signature = aggregateCommit.certificateSignature
-    networkIdentifier = network identifier of the chain
-    bftParams = bftModule.getBFTParameters(aggregateCommit.height)
-    validatorKeys = list of BLS public keys of validators in bftParams.validators obtained via
-                    validatorsModule.getValidatorAccount
-    sort validatorKeys lexicographically
-    weights = array of validator BFT weights obtained from bftParams.validators such that
-              weights[i] is the BFT weight of the validator with the BLS key validatorKeys[i]
```

Comment:
Does it make any difference if we sort `validatorKeys` lexicographically here or we do it in `verifyAggregateCertificateSignature`? or is it just more suitable to be in `verifyAggregateCertificateSignature`

Reply:
In terms of protocol, it is the same. The reason I changed it in the specifications is that in `aggregateSingleCommits` we also do the sorting within the function and I think it is more robust to sort within the function than to assume that the input parameter is sorted in a certain way.